PROJECT.md

```diff
@@ -2,34 +2,34 @@
 
 Each project repository **requires**, at a minimum:
 
-1. Readme -- `README.md`
-1. License -- `LICENSE`
-1. Contributing Guidelines -- `.github/CONTRIBUTING.md`
-1. Issue Template -- `.github/ISSUE_TEMPLATE.md`
+1. Readme — `README.md`
+1. License — `LICENSE`
+1. Contributing Guidelines — `.github/CONTRIBUTING.md`
+1. Issue Template — `.github/ISSUE_TEMPLATE.md`
 
-## Readme
+## Readme File
 
-Each **readme** follows a [`README.md` template](./templates/README.md)
+Each [README](https://en.wikipedia.org/wiki/README) file follows a [`README.md` template](./templates/README.md).
```

Comment:
Could we follow the convention of bolding the "readme" in this section as we have below?